### PR TITLE
Support multiple playbooks for provisioning

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,4 @@ Gemfile.lock
 .ruby-gemset
 *.retry
 *.pyc
+user_playbooks
```
Comment (akofink) [inline .gitignore]: We should go ahead and create this directory either with a `.gitkeep` file or with an example playbook.

Comment (ehelms) [inline .gitignore]: Does gitkeep prevent you from committing anything you add to it? I want to help prevent devs accidentally committing their custom scripts.

Comment (akofink) [inline .gitignore]: It's just a way to commit an empty directory to a git repo. You could also ignore `user_playbooks/*.yml` and add a `user_playbooks/playbook1.yml.example`. I just don't want our users to have to create the `user_playbooks` folder.

**Comment by ehelms** [comment]
Updated @akofink

File: Vagrantfile [hunk 2]
```diff
@@ -64,11 +64,25 @@ module Forklift
...
+          if playbooks.is_a?(String)
+            machine.vm.provision 'main', type: 'ansible' do |ansible|
+              ansible.playbook = playbooks
+
+              ansible.groups = @ansible_groups
+            end
+
+          elsif playbooks.is_a?(Array)
+
+            playbooks.each do |playbook, index|
```
Comment (zjhuntin) [inline Vagrantfile]: I think this needs to be playbooks.each_with_index

**Comment by akofink** [comment]
Very nice. ACK

**Comment by ehelms** [comment]
@zjhuntin updated -- I think it was just working because I kept redefining the main provisioner, using an acutal index is probably safer in the long run.

**Comment by zjhuntin** [comment]
Works great! ACK

**Pull request merged**